contracts/governance/PausableGuardian_0_8.sol

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2017-2022, OokiDao. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-4.3.2/access/Ownable.sol";
+
+
+contract PausableGuardian_0_8 is Ownable {
```

Comment:
I think for future extensibility we have to use ROLES from openzeppelin. I have used them and they are much more extensible. in next protocol version I think we could get them  used. just fyi

Reply:
actually might be a good idea to do them now here. 
```
import "@openzeppelin/access/AccessControl.sol";

contract Asdf is Ownable, AccessControl {
bytes32 public constant SOME_ROLE = keccak256("SOME_ROLE");
    modifier onlyHasRole(bytes32 _role) {
        require(
            hasRole(_role, _msgSender()),
            "onlyHasRole: msg.sender does not have role"
        );
        _;
    }
    function mint(address _to, uint256 _amount)
        public
        onlyHasRole(SOME_ROLE)
    {
     ...
    }
}
```

if we will employ this strategy in the future in the protocol. wrapping around a module. we would be able to have couple of roles e.g
1) guardian role
2) SWEEP_Role
.. and maybe others